lists/eg.csv

```diff
@@ -56,15 +60,19 @@ http://misrdigital.blogspirit.com,NEWS,News Media,2014-04-15,citizenlab,
 http://misrdigital.blogspot.com,NEWS,News Media,2014-04-15,citizenlab,
 http://missmabrouk.blogspot.com,NEWS,News Media,2014-04-15,citizenlab,
 http://mostafa.foolab.org,NEWS,News Media,2014-04-15,citizenlab,
+http://www.moheet.com,NEWS,News Media,2017-05-28,ADEF,blocked as of date added
 http://rassd.com,NEWS,News Media,2014-04-15,citizenlab,
 http://samy.attahawy.net,NEWS,News Media,2014-04-15,citizenlab,
 http://seekingfreedom.blogspot.com,NEWS,News Media,2014-04-15,citizenlab,
+http://www.sinaihr.org,HUMR,Human Rights Issues,2017-05-28,ADEF,blocked as of date added
 http://tadwen.com,NEWS,News Media,2014-04-15,citizenlab,
 https://textsecure-service-ca.whispersystems.org:80,COMT,Communication Tools,2017-02-10,citizenlab,Used by Signal Desktop for communication
 https://textsecure-service-ca.whispersystems.org:4433,COMT,Communication Tools,2017-02-10,citizenlab,Used by Signal Desktop for communication
 https://textsecure-service-ca.whispersystems.org:8443,COMT,Communication Tools,2017-02-10,citizenlab,Used by Signal Desktop for communication
 https://whispersystems-textsecure-attachments.s3.amazonaws.com,COMT,Communication Tools,2017-02-10,citizenlab,Used by Signal desktop for attachments
 http://theegyptblog.blogspot.com,NEWS,News Media,2014-04-15,citizenlab,
+https://torproject.org,ANON,Anonymization and circumvention tools,2017-05-27,ADEF,blocked as of date added
```

Comment:
This link is already in the [global list](https://github.com/citizenlab/test-lists/blob/master/lists/global.csv#L981).
In ooniprobe the global list is always tested regardless of the localization.